### PR TITLE
Set PPT mode to AUTO (1)

### PR DESCRIPTION
**Pull request by shihaocao**
# Set PPT mode to AUTO (1)

Note:
This PR no longer has changes related to forcing a loop to trigger the watchdog timer since the reset command seems to fully reset the spacecraft succesfully.

### Changes
- Change the default ppt mode to auto and max tracking
- Change the unit test to accept an FSW default

### Testing
Tested on 02/04: POC Shihao Cao

### CODE REVIEW

**Review by kylekrol — CHANGES_REQUESTED** [review]
Changes look good to me; although, it seems that we need to run some unit tests locally to fix CI. I'm sure we broke a couple HOOTL, HITL, and unit tests.

File: src/fsw/FCCode/GomspaceController.hpp [hunk 1]
```diff
@@ -8,7 +8,7 @@
 #include <common/constant_tracker.hpp>
 
 TRACKED_CONSTANT_SC(unsigned int, default_pv_cmd, 4000);
-TRACKED_CONSTANT_SC(unsigned char, default_ppt_mode, 2);
+TRACKED_CONSTANT_SC(unsigned char, default_ppt_mode, 1);
```
Comment (kylekrol) [inline src/fsw/FCCode/GomspaceController.hpp:11]: Wooo!

**Comment by shihaocao** [comment]
Athena pointed out that if we went into this while loop while the inner tank was venting to the outer tank, we'd definitely overpressurize and explode. 

She suggested that instead of using a while loop, we just implement a "mute" on GomSpace behavior

**Comment by kylekrol** [comment]
Ah okay fair enough. So something like if the reboot flag is true then the GOMSpace control task just doesn't execute any code?

File: src/fsw/FCCode/GomspaceController.cpp [hunk 2]
```diff
@@ -191,6 +191,10 @@ GomspaceController::GomspaceController(StateFieldRegistry &registry, unsigned in
      }
 
 void GomspaceController::execute() {
+    // Prevent watchdog timer petting
+    if(gs_reset_cmd_f.get())
+        return;
```
Comment (shihaocao) [inline src/fsw/FCCode/GomspaceController.cpp:196]: doing it this way still allows other control tasks to attempt to continue working, notably, PropSystem still maintains the ability to close the intertank valve.

File: src/fsw/FCCode/GomspaceController.cpp [hunk 3]
```diff
@@ -211,8 +212,6 @@ void GomspaceController::execute() {
         pv2_output_cmd_f.set(gs.hk->vboost[1]);
         pv3_output_cmd_f.set(gs.hk->vboost[2]);
 
-        ppt_mode_cmd_f.set(gs.hk->pptmode);
```
Comment (shihaocao) [inline src/fsw/FCCode/GomspaceController.cpp:214]: removing this, so that FlightSoftware still has authority to set a default

**Comment by shihaocao** [comment]
I removed the reboot loop because the reset command fully resets the spacecraft. Thus we should just not use the reboot command, and only the reset command.

**Review by fatimayousuf — APPROVED** [review]

**Review by kylekrol — APPROVED** [review]

**Pull request merged**